### PR TITLE
Update setup.nsi

### PR DESCRIPTION
**Pull request by Sami32**
Some renderers was forgotten.

### CODE REVIEW

File: src/main/external-resources/nsis/setup.nsi [hunk 1]
```diff
@@ -372,37 +377,44 @@ Section "Uninstall"
 	Delete /REBOOTOK "$INSTDIR\renderers\Microsoft-Xbox360.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Microsoft-XboxOne.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Miracast-M806.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\Netgear-NeoTV.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Naim-Mu-So.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\etgear-NeoTV.conf"
```
Comment (SubJunk) [inline src/main/external-resources/nsis/setup.nsi]: Forgot the N

File: src/main/external-resources/nsis/setup.nsi [hunk 2]
```diff
@@ -372,37 +377,44 @@ Section "Uninstall"
...
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraTHPU30Z.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraTXL32V10E.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraVT60.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Philips-AureaAndNetTV.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Philips-PFL.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Philips-Streamium.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Pioneer-BDP.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Pioneer-Kuro.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\PopcornHour.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\README.txt"
```
Comment (SubJunk) [inline src/main/external-resources/nsis/setup.nsi]: Why?

File: src/main/external-resources/nsis/setup.nsi [hunk 3]
```diff
@@ -344,20 +344,25 @@ Section "Uninstall"
...
 	; Current renderer files
 	Delete /REBOOTOK "$INSTDIR\renderers\AnyCast.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Apple-TV-VLC.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice-AirPlayer.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice-VLC32bit.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\BlackBerry-PlayBook-KalemSoftMP.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\CambridgeAudio-AzurBD.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\DLink-DSM510.conf"
```
Comment (SubJunk) [inline src/main/external-resources/nsis/setup.nsi]: "DL" shouldn't come before "De" in my opinion. I know some systems like to treat capitals differently but I don't find that as intuitive.

Comment (Sami32) [inline src/main/external-resources/nsis/setup.nsi]: No problem, i just copied the Github order.

File: src/main/external-resources/nsis/setup.nsi (hunk 2 above)
Comment (Sami32) [inline src/main/external-resources/nsis/setup.nsi]: Because i copy paste all the renderer only and forgotten that it had be deleted...

**Comment by Sami32** [comment]
@SubJunk Done.

**Pull request merged**